errbot/backends/test.py

```diff
@@ -413,7 +413,20 @@ def test_about(self, testbot):
     def on_finish():
         bot.stop()
 
+    #  setup the logging to something digestable.
+    import logging
```

Comment:
`logging` is already imported at the top of the file, so this import should be removed here I think.